app/src/main/java/io/homeassistant/companion/android/widgets/button/ButtonWidgetConfigureActivity.kt

```diff
@@ -454,6 +455,10 @@ class ButtonWidgetConfigureActivity : BaseWidgetConfigureActivity() {
                 ButtonWidget.EXTRA_ICON_NAME,
                 binding.widgetConfigIconSelector.tag as String
             )
+            intent.putExtra(
+                ButtonWidget.EXTRA_ICON_COLOR,
+                binding.iconColor.text.toString()
+            )
```

Comment:
Please add validation to prevent invalid input (see crash mentioned). If blank, the value should not be sent to avoid saving unnecessary blank text in the database.